### PR TITLE
[Magic Transit] Added UI steps for creating static routes

### PR DESCRIPTION
**Pull request by dcpena**
- Added steps for adding a static route via the dashboard.
- Removed partial for creating/editing static routes via the API because it only appears in one place now.

### CODE REVIEW

File: products/magic-transit/src/content/get-started/configure-tunnels/assign-tunnel-route-priorities.md [hunk 1]
```diff
@@ -88,9 +86,25 @@ The minimum advertising prefix is /24, but because Cloudflare uses GRE tunnels a
 
 For example, you can send `x.x.x.0/29` to Datacenter 1 and `x.x.x.8/29` to Datacenter 2. This is helpful when you operate in an environment with constrained IP resources.
 
-## Create and edit static routes
+## Static routes
+
+Magic Transit uses the static routes you provide to route traffic through GRE tunnels. A route with a lower **Priority** value is used as the preferred route, and routes with the same priority value use equal-cost multi-path (ECMP) packet forwarding to route traffic.
```
Comment (kodster28) [inline products/magic-transit/src/content/get-started/configure-tunnels/assign-tunnel-route-priorities.md:91]:
```suggestion
Magic Transit uses the static routes you provide to route traffic through GRE tunnels. A route with a lower **Priority** value is used as the preferred route and routes with the same priority value use equal-cost multi-path (ECMP) packet forwarding to route traffic.
```

File: products/magic-transit/src/content/get-started/configure-tunnels/assign-tunnel-route-priorities.md [hunk 2]
```diff
@@ -88,9 +86,25 @@ The minimum advertising prefix is /24, but because Cloudflare uses GRE tunnels a
...
+You can also create and edit static routes using [Magic Transit Static Routes API](https://api.cloudflare.com/#magic-transit-static-routes-properties).
+
+### Create a static route
+
+1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login) and select **Magic Transit**.
+1. Next to **Static routes configuration**, click **Configure**.
+1. On the **Static Routes** page, click **Create** to add a new route.
+1. Enter the information for your route and click **Add routes** when you are done.
+
+While optional, we highly recommend testing your route before adding it by clicking **Test routes**.
```
Comment (kodster28) [inline products/magic-transit/src/content/get-started/configure-tunnels/assign-tunnel-route-priorities.md]: Should you move this up before the last step?

**Review by kodster28 — APPROVED** [review]

File: products/magic-transit/src/content/get-started/configure-tunnels/assign-tunnel-route-priorities.md (hunk 2 above)
Comment (dcpena) [reply products/magic-transit/src/content/get-started/configure-tunnels/assign-tunnel-route-priorities.md]: Good call. Done

File: products/magic-transit/src/content/get-started/configure-tunnels/assign-tunnel-route-priorities.md (hunk 1 above)
Comment (dcpena) [reply products/magic-transit/src/content/get-started/configure-tunnels/assign-tunnel-route-priorities.md:91]: The nerd in me is having a hard time removing the comma to join a compound sentence. I'm keeping it 😭

**Pull request merged**